docs/api/index.md

```diff
@@ -1,5 +1,7 @@
 DDK API works with WebSockets via [Socket.IO](https://socket.io/) library
```

Comment:
You should provide socket.io-client library too in order to communicate with our API.

Reply:
I did not understand